src/types/events.ts

```diff
@@ -158,6 +160,36 @@ export class SmartContractModuleContractCreatedEvent {
     assert(this.isV105)
     return this._chain.decodeEvent(this.event)
   }
+
+  /**
+   * A contract got created
+   */
+  get isV147(): boolean {
+    return this._chain.getEventHash('SmartContractModule.ContractCreated') === '7b1fac3eee26bad6f35e60f95445eaff64030725c074c795899e96ed831a51bc'
+}
```

Comment:
wrong indentation here

Reply:
Yes, I noticed that this file uses 2 spaces. I left it intact so it is easier to spot the changes I made.
Should I reformat the whole file?